alcs-frontend/src/app/features/application/applicant-info/applicant-info.component.ts

```diff
@@ -36,8 +37,18 @@ export class ApplicantInfoComponent implements OnInit, OnDestroy {
         this.fileNumber = application.fileNumber;
 
         this.submission = await this.applicationSubmissionService.fetchSubmission(this.fileNumber);
-        this.isSubmitted =
-          application.source === APPLICATION_SYSTEM_SOURCE_TYPES.APPLICANT ? !!application.dateSubmittedToAlc : true;
+        const isApplicantSubmission = application.source === APPLICATION_SYSTEM_SOURCE_TYPES.APPLICANT;
+
+        this.isSubmitted = isApplicantSubmission ? !!application.dateSubmittedToAlc : true;
+
+        this.wasSubmittedToLfng =
```

Comment:
I think this will still cause issues, if the application is now in Under Review by ALC or any future statuses this would no longer be editable. This flag does not accurately represent its name.

Reply:
Future statuses are handled by `.isSubmitted`, and I just tested 'Under Review by ALC' and 'Decision Released' and they work as expected!

Reply:
Updated variable name to clarify status